src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/api/TemplateAnotation.kt

```diff
@@ -0,0 +1,4 @@
+package org.vaccineimpact.orderlyweb.controllers.api
```

Comment:
Filename typo 'Anotation'. Also, shouldn't this be in the 'web' dir under controllers?

Reply:
yes it should!